Pyrado/pyrado/algorithms/step_based/actor_critic.py

```diff
@@ -95,13 +96,13 @@ def expl_strat(self) -> NormalActNoiseExplStrat:
         return self._expl_strat
 
     @property
-    def sampler(self) -> ParallelRolloutSampler:
+    def sampler(self) -> SamplerBase:
         return self._sampler
 
     @sampler.setter
-    def sampler(self, sampler: ParallelRolloutSampler):
-        if not isinstance(sampler, (ParallelRolloutSampler, CVaRSampler)):
-            raise pyrado.TypeErr(given=sampler, expected_type=(ParallelRolloutSampler, CVaRSampler))
+    def sampler(self, sampler: SamplerBase):
+        if not isinstance(sampler, (SamplerBase)):
+            raise pyrado.TypeErr(given=sampler, expected_type=SamplerBase)
```

Comment:
We could also use a Union here. I replaced this with `SamplerBase` in this case, because no special methods of `RolloutSavingWrapper` are used in this class. This reduces complexity and also makes adding new samplers easier in the future.